eZ/Publish/Core/QueryType/QueryTypeMapper.php

```diff
@@ -0,0 +1,13 @@
+<?php
+/**
+ * This file is part of the ezplatform package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\Core\QueryType;
+
+interface QueryTypeMapper
+{
+    public function map($input);
```

Comment:
Not if I want it to be an interface that maps something _to_ a `QueryType`. From this interface's perspective, all we know is that we map something to a QueryType. I'll add phpdoc for the method and class, though.

I initially had a typehint on `ContentView` (since it is what the implementation maps on), but then the interface would have been a `ContentViewQueryTypeMapper`.